.github/workflows/lint.yml

```
@@ -11,13 +11,13 @@ jobs:
     steps:
       - uses: actions/setup-go@v3
```

```suggestion
      - uses: actions/setup-go@v4
```

According to latest docs